### DISCUSSION TITLE
Using NetworkOnly for api requests breaks all caching

### DISCUSSION THREAD

**Discussion by Yonisegev** [shadowwalker/next-pwa#464] [Q&A]
Hey,
I'm using:
next 13.1.1
next-pwa 5.6.0

With a very simple PWA configuration:

```
const withPWA = require('next-pwa')({
    dest: 'public',
    cacheOnFrontEndNav: true,
    runtimeCaching: [{
        urlPattern: /(notify|visible)/,
        handler: 'NetworkOnly',
    }]
})

module.exports = withPWA(nextConfig)
```

When running the PWA without the runtimeCaching configuration, everything is cached as needed and offline mode is working fine.

When adding the above runtimeCaching configuration (which whole purpose is I don't want the SW to give a response from the cache instead of doing a network call, because it's an analytic event call),
The files are still cached as needed but when toggling offline mode and refreshing I get the browser error indicating there's no internet.)

I also tried doing it with a custom worker like so (simple implementation here):
worker/index.js:
```
self.addEventListener('fetch', (e) => {
    if (e.request.url.includes('visible')) {
        return false;
    }
})

```

But I still get the same thing.

I actually see the sw.js file in the browser having my configuration appended to it:
`e.registerRoute(/(notify|visible)/, new e.NetworkOnly, "GET")
`
Any idea how to solve this?

**Comment by ImBIOS** [comment]
@Yonisegev It seems this repo is dead and the maintainer cannot be contacted for so long now. I've made an immediate response by resurrecting this repo [ImBIOS/next-pwa](https://github.com/ImBIOS/next-pwa).
